contracts/core/liquidators/twap-impl/TWAPAuction.sol

```diff
@@ -1,7 +1,8 @@
 /*
     Copyright 2020 Set Labs Inc.
 
-    Licensed under the Apache License, Version 2.0 (the "License");
+    Licensed under the Apache License,
```

Comment:
Why 2 lines?

Reply:
My command key has been screwy haha, must have accidentally done that when I was trying to hot key something